junebug/stores.py

```diff
@@ -0,0 +1,53 @@
+from twisted.internet.defer import inlineCallbacks, returnValue
+from vumi.message import TransportUserMessage
+
+
+class BaseStore(object):
+    '''Base class for store objects. Stores data in redis in a hash.
+    redis: redis manager
+    ttl: expiry for keys in the store
```

Comment:
The standard seems to be to put it on the class and to leave the docstring
for __init__ empty.